packages/vhd-lib/src/Vhd/VhdAbstract.js

```diff
@@ -164,4 +167,44 @@ export class VhdAbstract {
       }
     }
   }
+
+  static async rename(handler, sourcePath, targetPath) {
+    await handler.rename(sourcePath, targetPath)
+  }
+
+  static async unlink(handler, path) {
+    const resolved = await resolveAlias(handler, path)
+    try {
+      await handler.unlink(resolved)
+    } catch (err) {
+      if (err.code === 'EISDIR') {
+        await handler.rmtree(resolved)
+      } else {
+        throw err
+      }
+    }
+
+    // also delete the alias file
+    if (path !== resolved) {
+      await handler.unlink(path)
+    }
+  }
+
+  static createAlias(handler, aliasPath, targetPath) {
+    if (!isVhdAlias(aliasPath)) {
+      throw new Error(`Alias must be named *.alias.vhd,  ${aliasPath} given`)
+    }
+    if (isVhdAlias(targetPath)) {
+      throw new Error(`Chaining alias is forbidden ${aliasPath} to ${targetPath}`)
+    }
+    const aliasDir = path.resolve('/', path.dirname(aliasPath))
+
+    // handle case where target path is already a relative path from alis
+    const resolvedPathToTarget = path.resolve(aliasDir, targetPath)
+
+    // only store the relative path from alias to target
+    const relativePathToTarget = path.relative(aliasDir, resolvedPathToTarget)
+
+    return handler.writeFile(aliasPath, Buffer.from(relativePathToTarget))
```

Comment:
1. you can directly write a string
2. contrary to most of the legacy XO code: `await` instead of `return`
   - better stacktraces in modern engines
   - don't leak potential return values to the public API

```suggestion
    await handler.writeFile(aliasPath, relativePathToTarget)
```